source/dashboards/index.rst

```diff
@@ -48,10 +48,12 @@ Prometheus Dashboards
 *********************
 
 .. toctree::
-   :glob:
-   :maxdepth: 1
+   :maxdepth: 2
```

Comment:
This is duplicated content (also listed under insight). It's visible in HTML but not in PDF.